multiversx_sdk_cli/cli_shared.py

```diff
@@ -268,8 +269,15 @@ def send_or_simulate(tx: ITransaction, args: Any, dump_output: bool = True) -> C
             transaction_on_network = send_and_wait_for_result(tx, proxy, args.timeout)
             output_builder.set_awaited_transaction(transaction_on_network)
         elif send_only:
-            hash = proxy.send_transaction(tx)
-            output_builder.set_emitted_transaction_hash(hash)
+            try:
+                hash = proxy.send_transaction(tx)
+                output_builder.set_emitted_transaction_hash(hash)
+            except GenericError as ge:
+                url = ge.url
+                message = ge.data["error"]
+                data = ge.data["data"]
+                code = ge.data["code"]
```

Comment:
Just in case, perhaps use `ge.get("...", "")`?

Reply:
Done